packages/ai/core/embed/embed.ts

```diff
@@ -129,7 +129,7 @@ Only applicable for HTTP-based providers.
             return {
               embedding,
               usage,
-              rawResponse: modelResponse.rawResponse,
+              rawResponse: modelResponse.response,
```

Comment:
We need to consider renaming this on the `embed` method returns as well, and potentially for `embedMany`

Reply:
done!